lib/IDE/TypeReconstruction.cpp

```diff
@@ -1254,6 +1259,28 @@ static void VisitNodeFunction(
     }
   }
 
+  do {
+    if (cur_node->getKind() == Demangle::Node::Kind::Subscript) {
+      FindNamedDecls(ast, DeclBaseName::createSubscript(),
+                     decl_scope_result);
+      if (decl_scope_result._decls.empty()) {
+        result._error = stringWithFormat(
+          "subscript identifier could not be found by name lookup");
+        break;
```

Comment:
If we get to this point do we really want to continue on to the code at line 1303?

Reply:
See above